src/nCrCatalan/index.js

```diff
@@ -0,0 +1,18 @@
+/* Nth Catalan Number
+ * Function: nCrCatalan()
+ */
+ const validate = require('../validation/non-negative-integer');
+ const nCr = require('../nCr/index.js');
+ function nCrCatalan(num)
+{
+    if (num <= 1){return 1;}
```

Comment:
This much is all that is needed to calculate Catalan number, please read about it.
```js
function catalan(num){
    return nCr(2*num, num);
}
```